docker-compose.yml

```diff
@@ -34,3 +34,9 @@ services:
       - POSTGRES_USER=pontoon
       - POSTGRES_PASSWORD=asdf
       - POSTGRES_DB=pontoon
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+
+# Persist database
+volumes:
+  pgdata: null
```

Comment:
Post-merge question, would it be a good idea to use a volume name that's version-specific?

With the data migration we're doing now to 13, would that help a local update?

Reply:
I think in any case you need to be aware of what's happening when applying a Postgres version update, so I'm not really sure that it would really help. Maybe reconsider this when applying the next update, which in any case is the next time when it could make sense to change the volume name?

Reply:
Yeah. Background on that from the pontoon chat:

`docker-compose build` doesn't update the postgres container, that's only done once you `up`. At which point the new version of postgres is likely gonna try to do something with the old data and get folks and nervous and stuff. If at that point, it had a new volume, it'd just not find the old data, and you could import it.

It might also help to work on branches before and after the pg update. Even though the data won't sync, it might stick around? Not sure how graceful dc is about dangling volumes.